scripts/esbuild/mock-doc.ts

```diff
@@ -50,7 +50,6 @@ export async function buildMockDoc(opts: BuildOptions) {
     bundle: true,
     alias: mockDocAliases,
     logLevel: 'info',
-    target: 'node16',
```

Comment:
I'm not sure, it's possible there was some reason I had this where it was emitting JS that didn't work with node 16 _without_ it but I fear whatever justification there was may have been lost to the sands of time (we're talking about 3 months ago here)

Reply:
if it works well on our supported version without this I have no concern about removing it

Reply:
I created the following function:

```ts
export function getEsbuildTargets(): string[] {
  return ['node16', 'chrome79', 'edge79', 'firefox70', 'safari14'];
}
```
to ensure we set the targets based on what we need to support. I thought unifying the target property simplifies things but I can see that it might create confusion why we would set `node16` and other targets for something we only run in Node.js.

Happy to revert this change but it I think it doesn't change anything on the resulting mock-doc bundle.